src/dpl/include/dpl/Opendp.h

```diff
@@ -176,7 +176,7 @@ class Opendp
   Opendp(const Opendp&&) = delete;
   Opendp& operator=(const Opendp&&) = delete;
 
-  void init(dbDatabase* db, Logger* logger);
+  void init(dbDatabase* db, Logger* logger, std::function<std::unique_ptr<Graphics>(Opendp*, float, const odb::dbInst*)> make_graphics);
```

Comment:
I'm not clear that this is a sustainable pattern.  Are you going to do this with every dependency?

Reply:
Yes, but in general we should move to decouple gui from tool code, and opt for observers that the gui can subscribe to instead.

Reply:
Changed to a Strategy pattern instead.